nixos/modules/services/x11/window-managers/qtile.nix

```diff
@@ -1,25 +1,59 @@
-{ config, lib, pkgs, ... }:
+{ config, pkgs, lib, ... }:
 
 with lib;
 
 let
   cfg = config.services.xserver.windowManager.qtile;
+  qtile = cfg.package;
 in
 
 {
   options = {
-    services.xserver.windowManager.qtile.enable = mkEnableOption "qtile";
+    services.xserver.windowManager.qtile = {
+      enable = mkEnableOption "qtile";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.qtile;
+        defaultText = literalExpression "pkgs.qtile";
+        example = literalExpression "pkgs.unstable.qtile";
```

Comment:
I would remove this, only makes sense on stable.
```suggestion
```

Reply:
Ok, I just wanted to add an example, but it might confuse some users.